apps/web/pages/api/book/event.ts

```diff
@@ -508,7 +508,27 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       }
     }
 
-    return prisma.booking.create(createBookingObj);
+    /*Validate if there is any stripe_payment credential for this user*/
+    const eventTypePayment = await prisma.credential.findFirst({
+      where: {
+        type: "stripe_payment",
+        userId: users[0].id,
+      },
+      select: {
+        id: true,
+      },
+    });
+    /** If the stripe_payment credential exists then it creates the booking OR*/
+    /** If the stripe_payment credential does not exist but the eventTypes is
+     *   without payment then create the booking
+     */
```

Comment:
Use same comment code for all the comment
```suggestion
     /** eventType doesn’t require payment then we create a booking
     * OR
     * stripePaymentCredential is found and price is higher than 0 then we create a booking
     */
```